src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1167,6 +1174,7 @@ def forward(
 
 class OwlViTForObjectDetection(OwlViTPreTrainedModel):
     config_class = OwlViTConfig
+    main_input_name = "pixel_values"
```

Comment:
I believe this line caused test errors previously, will need to double check.

Reply:
by default I think the `main_input_name` is `input_ids`. Before when it was just text guided detection, the first param was `input_ids`, and then `pixel_values`. Now, since `input_ids` can be `None`, I made `pixel_values` as the first param, which is why the test case was failing. Hence this change

Reply:
That makes sense! I'd comment out the slow flags (tests with `@slow` decoraters are skipper) within `tests/test_modeling_common.py` to double check this.

You can run the tests from the root of transformers repo as follows:
```
# All tests
pytest tests/models/owlvit/test_modeling_owlvit.py
# Run only integration tests
pytest tests/models/owlvit/test_modeling_owlvit.py::OwlViTModelIntegrationTest
# Run a single test
pytest tests/models/owlvit/test_modeling_owlvit.py::OwlViTModelIntegrationTest::test_inference
```

Reply:
This is too much of a breaking change, even for a newly released model.